### DISCUSSION TITLE
misc-macros.cfg

### DISCUSSION THREAD

**Discussion by SV06PLUSNOOB** [bassamanator/Sovol-SV06-firmware#48] [Q&A]
Managed to in stall Klipper. But misc-macros.cfg has start settings. For instance temp extr/ mesh works in two steps. First 150/60. Later extruder goed to temp from slicer. Is there a way to turn this off ?

**Comment by bassamanator** [comment]
In the latest version, it's 3 steps, actually. This behaviour is desired for many reasons. Feel free to edit the macro for your own needs, however.

**Comment by bassamanator** [comment]
>OK i will have a look. Also i get some unjknown commands. m201 m203 m205 m205 in that order.They are not in the slicer. And i can't really find them in the cfg files ?

Those commands are for Marlin, not Klipper.
